src/main/resources/GoogleApps/DriveMacro.xml

```diff
@@ -361,28 +361,28 @@ To insert or change the current Google Drive document being embedded, you need t
 
 {{html clean==false}}
 &lt;form action="" method="get"&gt;
-Insert a Google Document: 
+Insert a Google Document:
```

Comment:
If this line is displayed for users then it needs to be translated.

Reply:
Mmmh. That will be in another commit.
Theres 3-4 pages of content that contain phrases for users and have not been translated at all.
They also need to be polished in their language (e.g. there's a confusion between import and embed). I'd like to push this to a later branch so that we focus on the configuration aspect now.

It seems that this macro got changed just because of automatic editing. Maybe I should remove it.